_config.yml

```diff
@@ -1,15 +1,40 @@
-name: Lime – An Open Source Text Editor
-markdown: redcarpet
-highlighter: pygments
-exclude: ['Rakefile', 'Gemfile', 'Gemfile.lock']
-
-paginate: 5
-paginate_path: blog/page:num
-
-nav:
-  - text: Blog
-    url: /blog/
-  - text: Contributing
-    url: /contribute.html
-  - text: Wiki
-    url: https://github.com/limetext/lime/wiki
+# site information
+title: Lime Text
+description: An Open Source Text Editor
+url: http://limetext.org
+
+# ruby gems
+gems:
+  - jekyll-feed
+  - jemoji
+
+# default layouts
+defaults:
+  - scope:
+      type: pages
+    values:
+      layout: page
+  - scope:
+      type: posts
+    values:
+      layout: post
+
+# blog post location
+permalink: /news/:title/
```

Comment:
Not that I think anyone links to us, but this will break any existing permalinks.